src/chef/orb.yml

```diff
@@ -264,7 +264,7 @@ jobs:
   cookstyle:
     description: Execute Cookstyle analyzer
     docker:
-      - image: chef/chefdk
+      - image: chef/workstation:0.15.20
```

Comment:
I'm not found of pinning a specific version.
Using the latest one lets us to face problems (eg switch to Chef 16) before they happen on servers.